aten/src/ATen/native/Activation.cpp

```diff
@@ -172,6 +172,19 @@ Tensor & relu_(Tensor & self) {
   return at::threshold_(self, 0, 0);
 }
 
+Tensor swish(const Tensor& self) {
+  return self.mul(self.sigmoid());
+}
+
+Tensor & swish_(Tensor& self) {
+  return self.mul_(self.sigmoid_());
+}
+
+Tensor swish_backward(const Tensor& grad, const Tensor& self) {
```

Comment:
Note that you don't need to make this an actual function in native_functions (that gets exposed into the `torch.*` namespace and that you need to document, etc)
Usually, I would add this function to [this file](https://github.com/pytorch/pytorch/blob/a78024476ba030b3439726eb96ab55a23a38a453/tools/autograd/templates/Functions.cpp) (remove the entry in native_function.yaml as well).
And you can leave the derivatives.yaml definition as-is and it will work just fine.

Reply:
fwiw functions with _backward don't get bound automatically.